### PR TITLE
Disjoint SmartGraphs

### PR DESCRIPTION
**Pull request by hkernbach**
General Documentation

### CODE REVIEW

File: 3.7/graphs-smart-graphs.md [hunk 1]
```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
 
+Benefits of DisjointSmartGraphs
+-----------------------
+
+DisjointSmartGraphs are a more specialized type of SmartGraphs. Compared to 
+SmartGraphs they are quite equal compared to their origin, expect one important
+detail:
+
+Where SmartGraphs are almost local, DisjointSmartGraphs are guaranteed to be
+local in any case by definition. A DisjointSmartGraph prohibids connected edges
```
Comment (joerg84) [inline 3.7/graphs-smart-graphs.md:82]: prohibits

File: 3.7/graphs-smart-graphs.md [hunk 2]
```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of DisjointSmartGraphs
```
Comment (joerg84) [inline 3.7/graphs-smart-graphs.md:74]: What is the spelling? I assumed DisjointSmartGraphs

Comment (hkernbach) [reply 3.7/graphs-smart-graphs.md:74]: DisjointSmartGraphs is correct.

File: 3.7/graphs-smart-graphs.md [hunk 3]
```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
+-----------------------
+
+DisjointSmartGraphs are a more specialized type of SmartGraphs. Compared to
```
Comment (markuspf) [inline 3.7/graphs-smart-graphs.md:77]:
```suggestion
DisjointSmartGraphs are a specialized type of SmartGraphs. 
```

File: 3.7/graphs-smart-graphs.md [hunk 4]
```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of DisjointSmartGraphs
+-----------------------
+
+DisjointSmartGraphs are a more specialized type of SmartGraphs. Compared to 
+SmartGraphs they are quite equal compared to their origin, expect one important
+detail:
```
Comment (markuspf) [inline 3.7/graphs-smart-graphs.md:79]:
```suggestion
```

File: 3.7/graphs-smart-graphs.md [hunk 5]
```diff
@@ -71,6 +71,20 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
+
+Where SmartGraphs are almost local, DisjointSmartGraphs are guaranteed to be
+local in any case by definition. A DisjointSmartGraph prohibits edges between
+different SmartGraph components. As no connections between subgraphs are
+existing, no network overhead will ever appear, which will lead to faster
+query traversal execution times. Besides that difference, the behaviour of
+all other functionalities are the same.
+
```
Comment (markuspf) [inline 3.7/graphs-smart-graphs.md:87]: In addition to the guaranteed sharding in SmartGraphs, a DisjointSmartGraph prohibits edges between vertices with different smartGraphAttribute values.

This ensures graph traversals, shortest path, and k-shortest-paths queries can be executed locally on a DBServer, achieving improved performance for this type of query.

File: 3.7/graphs-smart-graphs.md [hunk 6]
```diff
@@ -71,6 +71,18 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of DisjointSmartGraphs
+-----------------------
+
+DisjointSmartGraphs are a specialized type of SmartGraphs.
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:77]:
```suggestion
Disjoint SmartGraphs are a specialized type of SmartGraphs. 
```

File: 3.7/graphs-smart-graphs.md [hunk 7]
```diff
@@ -71,6 +71,18 @@ overhead. The more subgraphs are touched the more network cost will apply.
 However the overall performance is never worse than the same query using a
 General Graph.
 
+Benefits of DisjointSmartGraphs
+-----------------------
+
+DisjointSmartGraphs are a specialized type of SmartGraphs. 
+
+In addition to the guaranteed sharding in SmartGraphs, a DisjointSmartGraph
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:79]:
```suggestion
In addition to the guaranteed sharding in SmartGraphs, a Disjoint SmartGraph
```

File: 3.7/graphs-smart-graphs.md [hunk 8]
```diff
@@ -71,6 +71,18 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
+prohibits edges between vertices with different smartGraphAttribute values.
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:80]:
```suggestion
prohibits edges between vertices with different `smartGraphAttribute` values.
```

File: 3.7/graphs-smart-graphs.md [hunk 9]
```diff
@@ -71,6 +71,18 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
+
+This ensures graph traversals, shortest path, and k-shortest-paths queries
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:82]:
```suggestion
This ensures that graph traversals, shortest path, and k-shortest-paths queries
```

File: 3.7/graphs-smart-graphs.md [hunk 10]
```diff
@@ -71,6 +71,18 @@ overhead. The more subgraphs are touched the more network cost will apply.
...
+can be executed locally on a DB-Server, achieving improved performance for
+this type of query.
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:84]:
```suggestion
these type of queries.
```

File: 3.7/graphs-smart-graphs.md [hunk 11]
```diff
@@ -100,6 +112,24 @@ required and cannot be modified later.
 {% endarangoshexample %}
 {% include arangoshexample.html id=examplevar script=script result=result %}
 
+**Create a disjoint graph**
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:115]:
```suggestion
**Create a Disjoint SmartGraph**
```

File: 3.7/graphs-smart-graphs.md [hunk 12]
```diff
@@ -100,6 +112,24 @@ required and cannot be modified later.
...
+
+In contrast to regular SmartGraphs we have to add one option when creating the
+graph. The boolean option `isDisjoint` is required, needs to be set to `true`
+and cannot be modified later. 
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline smartGraphCreateGraphHowTo1_disjoint_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{smartGraphCreateGraphHowTo1_disjoint_cluster}
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:123]:
```suggestion
    @startDocuBlockInline smartGraphCreateGraphHowToDisjoint1_cluster
    @EXAMPLE_ARANGOSH_OUTPUT{smartGraphCreateGraphHowToDisjoint1_cluster}
```

File: 3.7/graphs-smart-graphs.md [hunk 13]
```diff
@@ -100,6 +112,24 @@ required and cannot be modified later.
...
+      var graph_module = require("@arangodb/smart-graph");
+      var graph = graph_module._create("myGraph", [], [], {smartGraphAttribute: "region", numberOfShards: 9, isDisjoint: true});
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph");
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock smartGraphCreateGraphHowTo1_disjoint_cluster
```
Comment (Simran-B) [inline 3.7/graphs-smart-graphs.md:129]:
```suggestion
    @endDocuBlock smartGraphCreateGraphHowToDisjoint1_cluster
```

**Pull request merged**